### DISCUSSION TITLE
`<Button><PlusIcon/></Button>` only supports a square shape.

### DISCUSSION THREAD

**Discussion by ramoneguru** [twilio-labs/paste#459] [General] [answered]
Paste seems to only support the square style for the `<Button> + <Icon>` section here: https://paste.twilio.design/components/button#icon-only-button

Is it possible to have a circular shape as well or will Paste only support this square style?

**Comment by SiTaggart** [comment, marked as answer]
For now, Paste is only supporting the button styles listed on the button document page.

We are actively moving away from the FAB button we see in console
 
![image](https://user-images.githubusercontent.com/368249/82934951-317a2300-9f41-11ea-9693-3d2640462a99.png)

If we see a need for a different shape button come up, outside of the FAB, we'll of course reconsider that position again. So if you have one, please share with the team or come to a Design Systems Office hours.
